assets/admin/listToolbarActions/AlertNamesToolbarAction.js

```diff
@@ -0,0 +1,30 @@
+import {translate} from 'sulu-admin-bundle/utils';
+import {AbstractListToolbarAction} from 'sulu-admin-bundle/views';
+
+export default class AlertNamesToolbarAction extends AbstractListToolbarAction {
+    getToolbarItemConfig() {
+        const {disable_for_empty_selection: disableForEmptySelection = false} = this.options;
+
+        return {
+            type: 'button',
+            label: translate('app.alert_names'),
+            icon: 'su-bell',
```

Comment:
Available icons are documented in the [Sulu Javascript Docs](https://jsdocs.sulu.io/latest/#icon).